railties/lib/rails/source_annotation_extractor.rb

```diff
@@ -168,6 +168,8 @@ def find(dirs)
     def find_in(dir)
       results = {}
 
+      return results if dir.match?(%r{(^|/)app/assets/builds(/|$)})
```

Comment:
I'm wondering if this should be made more generic, there might be other directories to exclude as well.

```suggestion
      return results if dir.in?(self.class.excluded_directories)
```

This would work similar to how directories are registered:
```ruby
    class Annotation < Struct.new(:line, :tag, :text)
      def self.excluded_directories
        @@excluded_directories ||= []
      end

      def self.register_excluded_directories(*dirs)
        excluded_directories.push(*dirs)
      end
    ...
```

Reply:
Also, this `app/assets/builds` is configured in js-bundling-rails:
https://github.com/rails/jsbundling-rails/blob/main/lib/install/install.rb#L2-L3
Rails itself doesn't know about this directory, especially with import maps being the default.
So js-bundling-rails should probably call:
```ruby
Annotation.register_excluded_directories("app/assets/builds")
```

Reply:
@p8 this looks more elegant. I've implemented the changes and will need to check how the wiring is set up from js-bundling-rails and create a PR there too. thanks Petrik!